WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPublishSettingsViewModel.kt

```diff
@@ -161,6 +171,41 @@ class EditPostPublishSettingsViewModel
         }
     }
 
+    fun onNotificationCreated(scheduleTime: SchedulingReminderModel.Period?) {
+        _onNotificationTime.value = scheduleTime
+    }
+
+    fun scheduleNotification(post: PostModel, notificationTime: SchedulingReminderModel.Period) {
+        updateNotifications(post, notificationTime)
+        updateUiModel(post)
+    }
+
+    private fun updateNotifications(
+        post: PostModel,
+        domainModel: SchedulingReminderModel.Period = OFF
```

Comment:
Wonder, was this one supposed to be called something different than `domainModel` (eg. `notificationTime`?)